HLTrigger/JSONMonitoring/plugins/TriggerJSONMonitoring.cc

```diff
@@ -529,7 +531,14 @@ TriggerJSONMonitoring::endLuminosityBlockSummary(const edm::LuminosityBlock& iLu
 void
 TriggerJSONMonitoring::globalEndLuminosityBlockSummary(const edm::LuminosityBlock& iLumi, const edm::EventSetup& iSetup, const LuminosityBlockContext* iContext, hltJson::lumiVars* iSummary)
 {
-  if (iSummary->processed->value().at(0)!=0) {
+
+  unsigned int iLs  = iLumi.luminosityBlock();
+  unsigned int iRun = iLumi.run();
+
+  evf::FastMonitoringService * fms_ = (evf::FastMonitoringService *)(edm::Service<evf::MicroStateService>().operator->());
```

Comment:
Please add a check for the availability of the FastMonitoringService or MicroStateService.
It is fine to print a LogError and/or disable some functionality if a service is not available, but it should not crash the whole program.

(actually, IMO *nothing* should ever crash the whole program, if a more graceful alternative is available)